nym-vpnd/src/command_interface/start.rs

```diff
@@ -100,17 +119,25 @@ pub(crate) fn start_command_interface(
     let uri_addr = default_uri_addr();
 
     let handle = std::thread::spawn(move || {
+        // Explicitly create a multu-threaded runtime for the command interface.
+        // We should consider also explictly setting the number of threads here to something like
+        // max(num_vcpu, 4) or something like that.
```

Comment:
that's a good point, should we perhaps deterine the vcpu split in `main.rs` and assign them accordingly to internal runtimes?

Reply:
It's an interesting challenge I've been pondering about, I'm leaning towards we should have more threads than vcpu. But it will depend, probabaly